js/classes/entities/foreground/hud.js

```diff
@@ -203,16 +250,16 @@ class AmmoLabel {
      * 
      * @param {Vector} pos the position of the AmmoLabel on the canvas
      * @param {number} type the ammo type associated with the AmmoLabel (should be a Projectile member type)
-     * @param {string} inputName the key used to select this ammo type
+     * @param {string} inputName the input key used to select this ammo type
      */
     constructor(pos, type, inputName) {
         this.type = type;
         this.label = new ItemLabel(
             pos, 
-            new Animator(Projectile.SPRITESHEET, new Vector(0, type * Projectile.SPRITESHEET_ENTRY_HEIGHT), 
-                Projectile.PROPERTY_TABLE[type].SIZE, 1, 1),
-            inputName,
-            INVENTORY.getAmmo(type).amount
+            new Animator(AmmoItem.SPRITESHEET, new Vector(0, type * AmmoItem.SPRITESHEET_ENTRY_HEIGHT), 
+                AmmoItem.SIZE, 1, 1),
```

Comment:
This should still be using `Projectile.PROPERTY_TABLE[type].SIZE` instead of `AmmoItem.SIZE`, since the former accounts for the empty pixels for smaller ammo types.